src/Microsoft.Graph/Generated/Sites/Item/Sites/Add/AddRequestBuilder.cs

```diff
@@ -0,0 +1,112 @@
+// <auto-generated/>
```

Comment:
Does it mean we can create a new site/subsite or this add/remove methods should be generated only for followed sites?

https://learn.microsoft.com/en-us/graph/api/site-follow?view=graph-rest-1.0&tabs=csharp

Reply:
I think you're correct here @MartinM85 This path should not be added as testing out on graph explorer leads to 400 errors as the `add` and `remove` methods should not be on the sites path. 
Created https://github.com/microsoftgraph/msgraph-metadata/issues/588 to address this